<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QPushButton" name="login_user_button">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="new_user_button">
       <property name="text">
        <string>New user</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QListView" name="user_list_view"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="remove_user_button">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QPushButton" name="schedule_stop_button">
       <property name="text">
        <string>Stop scheduler</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="schedule_start_button">
       <property name="text">
        <string>Start scheduled task</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="timer_display_button">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="search_button">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="working_time_button">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="export_result_button">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLineEdit" name="query_line"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>query_line</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>export_result_button</tabstop>
  <tabstop>schedule_start_button</tabstop>
  <tabstop>schedule_stop_button</tabstop>
  <tabstop>new_user_button</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>timer_display_button</tabstop>
  <tabstop>working_time_button</tabstop>
  <tabstop>user_list_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
